var/spack/repos/builtin/packages/py-torchmeta/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyTorchmeta(PythonPackage):
+    """A collection of extensions and data-loaders for few-shot learning &
+    meta-learning in PyTorch. Torchmeta contains popular meta-learning
+    benchmarks, fully compatible with both torchvision and PyTorch's DataLoader."""
+
+    homepage = "https://github.com/tristandeleu/pytorch-meta"
+    pypi     = "torchmeta/torchmeta-1.7.0.tar.gz"
+
+    version('1.7.0', sha256='148d42b6a1ec27970408f7bcb97cf1cb203f8699214e06424fe43d78faa848d9')
+
+    depends_on('python@3.6:',                   type=('build', 'run'))
+    depends_on('py-setuptools',                 type='build')
+    depends_on('py-numpy@1.14:',                type=('build', 'run'))
+    depends_on('py-torch@1.4.0:1.8.999',        type=('build', 'run'))
+    depends_on('py-torchvision@0.5.0:0.9.999',  type=('build', 'run'))
+    depends_on('py-pillow@7.0:',                type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('pil@7.0:',                      type=('build', 'run'))
```
Spack has a virtual dependency for this